src/utils/misc.cpp

```diff
@@ -155,9 +155,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath) {
                 QStringLiteral("nautilus"),
                 QStringList{"--no-desktop", QDir::toNativeSeparators(path)});
         } else if (output == QStringLiteral("caja-folder-handler.desktop")) {
+            const QDir dir = QFileInfo(path).absoluteDir();
```

Comment:
Which issue are those changes for?

Reply:
The Caja file manager error. This is an old commit.

Reply:
Why is it in your PR? :laughing:

Reply:
I don't understand why this line is new in upstream :confused: 
It was already merged days ago, wasn't it?

Reply:
It already is.

Reply:
Ah it is in master, not in develop.

Reply:
If you are talking about `pbek/QOwnNotes`, it's in both. master is only a rebase of develop at the time of the last release.

Reply:
this is because you are sending changes directly from your `develop`.
Instead do this:

First reverse the HEAD of your github repo to the last commit merged by pbek:
```sh
git push origin +'commit hash':develop
git fetch upstream && git merge upstream/develop
git push origin develop
```

Now you and main QON repo are in sync.

For every pull request, this is the general workflow

```sh
git checkout -b your-branch-name
....//make changes
git commit -m "commit message"
git push origin your-branch-name
//create pull request from 'your-branch-name' to 'pbek:develop'
...//pull request review and merged,
//get the latest changes,
git checkout develop
git fetch upstream && git merge upstream/develop
//repeat the procedure for every new pull request
```

Reply:
Thank you very much for the explanation!